README.md

```diff
@@ -40,6 +40,8 @@ let mut sim = turmoil::Builder::new().build();
 // register a host
 sim.host("server", || async move {
     // host software goes here
+
+    Err("Host software finished unexpectedly")?
```

Comment:
`Ok(())`?

Reply:
Yes, wanted to show a good default since it's almost always the case that it's undesired behavior, unless you're testing a shutdown. But will switch to `Ok(())` for brevity.